docker/docker-compose.tests.yml

```diff
@@ -9,6 +9,8 @@ services:
     build:
       context: ../
       dockerfile: docker/Dockerfile.test-database
+    environment:
+      MB_IMPORT_DUMPS: "false"
```

Comment:
In shell world, "false" is true. ;) (Any non-empty string is truthy.) This should be removed, since an undefined variable will do what you intend.